examples/get_latest_record.rs

```diff
@@ -0,0 +1,24 @@
+use streamstore::{
+    client::{ClientConfig, StreamClient},
+    types::{BasinName, ReadLimit, ReadRequest},
+};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let token = std::env::var("S2_AUTH_TOKEN")?;
+    let config = ClientConfig::new(token);
+    let basin: BasinName = "my-basin".parse()?;
+    let stream = "my-basin";
+    let stream_client = StreamClient::new(config, basin, stream);
+
+    let tail = stream_client.check_tail().await?;
+    let latest_seq_num = tail - 1;
```

Comment:
tail will be 0 if no records on the stream

Reply:
Oh yeah, I'll fix this.